src/test/test-math-util.c

```diff
@@ -54,7 +54,14 @@ TEST(iszero_safe) {
         assert_se( iszero_safe(DBL_MIN / -INFINITY));
         assert_se( iszero_safe(DBL_MAX / INFINITY / 2));
         assert_se( iszero_safe(DBL_MAX / -INFINITY * DBL_MAX));
+#if defined(__i386__)
```

Comment:
From the looks of it, `__SSE_MATH__` is defined if `-mfpmath=sse` is in effect, so maybe use that?

Though, shouldn't `iszero_safe` be fixed to deal with this (or declare `-mfpmath=sse` a requirement)?

Reply:
Thanks. Now also conditionalized with `__SSE_MATH__`.

> Though, shouldn't `iszero_safe` be fixed to deal with this (or declare `-mfpmath=sse` a requirement)?

At least currently, `iszero_safe()` is only used through `fp_equal()` in json.c. This is only used for parsing or formatting JSON. I do not think it is not necessary to request that all arithmetic give the same value in our code base, as we do not provide any numerical libraries.